test/testcases.md

```diff
@@ -69,14 +67,42 @@ This document will describe various test cases of the editor.js functionality. F
       - [ ] Checking the `shortcut` property
         - [ ] If `string` passed Editor.js should append the `tool` when such keys combination executed.
       - [ ] Checking the `inilineToolbar` property
-        - [ ] If `true`
-          - [ ] Editor.js should show the Inline Toolbar for this tool with [common](https://editorjs.io/configuration#inline-toolbar-order) settings.
-        - [ ] If `false`
-          - [ ] Editor.js should not show the Inline Toolbar for this tool.
-        - [ ] If `array`
-          - [ ] Editor.js should show the Inline Toolbar for this tool with a passed list of tools and their order.
-        - [ ] If omitted
-          - [ ] Editor.js should not show the Inline Toolbar for this tool.
+        - [ ] If `true` passed, the Editor.js should show the Inline Toolbar for this tool with [common](https://editorjs.io/configuration#inline-toolbar-order) settings.
+        - [ ] If `false` passed, the Editor.js should not show the Inline Toolbar for this tool.
+        - [ ] If `array` passed, the Editor.js should show the Inline Toolbar for this tool with a passed list of tools and their order.
+        - [ ] If omitted, the Editor.js should not show the Inline Toolbar for this tool.
       - [ ] Checking the `toolbox` property
         - [ ] If it contains `title`, this title should be used as a tool title
         - [ ] If it contains `icon`, this HTML code (maybe SVG) should be used as a tool icon
+
+- [ ] `onReady` property
+  - [ ] If `function` passed, the Editor.js should call the `function` when it's ready to work.
+  - [ ] If omitted, the Editor.js should be initialized with the `tools` only.
+
+- [ ] `onChange` property
+  - [ ] If `function` passed,the Editor.js should call the `function` when something changed in Editor.js DOM.
+  - [ ] If omitted, the Editor.js should be initialized with the `tools` only.
+
+- [ ] `data` property
+  - [ ] If omitted
+    - [ ] the Editor.js should be initialized with the `tools` only.
+    - [ ] the Editor.js should be empty.
+  - [ ] If `object` passed
+    - [ ] Checking the `blocks` property
+      - [ ] If `array` of `object` passed,
+        - [ ] for each `object` 
+          - [ ] Checking the `type` and `data` property
+            - [ ] the Editor.js should be initialize with `block` of class `type`
+            - [ ] If `type` not present in `tools`, the Editor.js should throw an error.
```

Comment:
in this case, we use the `Stub` tool. So we need to test it.

Reply:
Also, we need to check if the `data` property is passing to the tool's `render` method